code_pylint.py

```diff
@@ -18,15 +18,15 @@
 UNWATCHED_ERRORS = ['fixme', 'trailing-whitespace', 'import-error']
 
 MAX_ERROR_BY_TYPE = {
-                     'protected-access': 46,
+                     'protected-access': 41,
                      'invalid-name': 0,
                      'consider-using-f-string': 1,
                      'no-else-return': 0,
                      'arguments-differ': 2,
                      'no-member': 1,
                      'too-many-locals': 12,  # Reduce by dropping vectored objects
-                     'wrong-import-order': 7,
-                     'too-many-branches': 12,
+                     'wrong-import-order': 1,
+                     'too-many-branches': 10,
```

Comment:
Why did you change it ?
Is it a modification from a previous modification or is it to do the best as possible ?

If it is to do the best as possible, please don't modify this file, unless you have good reasons (which are that you have too many arguments or other problems like that, which can't be simply resolved except by changing the whole philosophy of your developments).

Reply:
It is from master because I did a merge from master. I Needed the update from master to have cadview